README.md

```diff
@@ -16,6 +16,8 @@ It requires:
 
 # Development
 
+## DO NOT USE STANDARD RAILS S
+
 To run locally you should run yarn
 
 ```
```

Comment:
I don't understand this. I see you have comments on the issue and if you made the Figma design it is a very nice job.
So please delete this branch and the pull request will go, as we don't need it.